### PR TITLE
feat(issues): Add Hydration Error resource to issue details

### PR DESCRIPTION
**Pull request by scttcper**
related answers post doesn't seem to have deployed yet https://github.com/getsentry/static-sites/pull/2183

### CODE REVIEW

File: static/app/utils/issueTypeConfig/errorConfig.tsx [hunk 1]
```diff
@@ -55,6 +55,11 @@ const ErrorInfoChecks: Array<ErrorInfo> = [
     projectCheck: true,
     errorHelpType: ErrorHelpType.DYNAMIC_SERVER_USAGE,
   },
+  {
+    errorTitle: 'Hydration Error',
```
Comment (scttcper) [inline static/app/utils/issueTypeConfig/errorConfig.tsx:59]: title doesn't really work here, will need to look at messages

Comment (scefali) [reply static/app/utils/issueTypeConfig/errorConfig.tsx:59]: @scttcper ok so you have to update this PR so it also looks at the message?

Comment (scttcper) [reply static/app/utils/issueTypeConfig/errorConfig.tsx]: i lied the title seems to contain it, just need to match the multiple options

Comment (scefali) [reply static/app/utils/issueTypeConfig/errorConfig.tsx]: @scttcper easy to reuse the same `errorHelpType`

**Review by scefali — APPROVED** [review]

**Review by roggenkemper — APPROVED** [review]

**Pull request merged**